### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by emanKhaledh**

### CODE REVIEW

File: src/controllers/index.js [hunk 1]
```diff
@@ -3,17 +3,26 @@ const router = express.Router();
 const insertProject = require("./insertProject");
 const home = require("./home");
 const signup = require("./signup");
+const login = require('./login');
+const project = require('./singleProject');
+const checkuser = require('./checkUser')
 
 router.get("/", home.get);
-router.post("/", home.post);
+// router.post("/", home.post);
```
Comment (yahyaHB) [inline src/controllers/index.js]: what do you use `post` method for??
and what could you use it for from the homepage?

File: src/controllers/index.js [hunk 2]
```diff
@@ -3,17 +3,26 @@ const router = express.Router();
...
 router.get("/signup", signup.get);
 router.post("/signup", signup.post);
 
-const expressValidator = require("express-validator");
-router.use(expressValidator());
+router.get('/login', login.get);
+router.post('/login', login.post);
+ router.get('/project/:id', project.get)
+
+// const expressValidator = require("express-validator");
```
Comment (yahyaHB) [inline src/controllers/index.js]: why you're not using the express validator?

File: src/controllers/home.js [hunk 3]
```diff
@@ -1,45 +1,41 @@
 
-const bcrypt = require('bcrypt');
-const   getUser = require('./../database/queries/get_login');
+// const bcrypt = require('bcrypt');
+// const getUser = require('./../database/queries/get_login');
```
[the remark this answers is not in the document]
Comment (ahmadmshatat) [reply src/controllers/home.js]: the login transferred from home page to another route

File: config.env [hunk 4]
```diff
@@ -1,2 +1,3 @@
 DB_URL = postgres://eman05:12345@localhost:5432/freelance
-SECRET= eman 
\ No newline at end of file
+
+SECRET = 'poiugyfguhijokpkoihugyfyguhijo';
```
[the remark this answers is not in the document]
Comment (ahmadmshatat) [reply config.env]: fixed in the latest pull

File: public/js/dom.js [hunk 5]
```diff
@@ -2,39 +2,57 @@ const select = function(selector){
...
+   
+
+   fetch('/login','POST' ,name, password, function(res){
+     console.log("resjjjj",res);
+     
      msg.textContent=res;
-      console.log("ee")
+      
      if(res === "success"){
+       console.log("go to home");
```
[the remark this answers is not in the document]
Comment (emanKhaledh) [reply public/js/dom.js]: ok

**Pull request closed without merging**